src/applications/facility-locator/sass/facility-locator.scss

```diff
@@ -677,17 +683,17 @@
 
 
 #infoBubble {
-  background-color: #e1f3f8;
+  background-color: $color-aqua-lightest;
   color: inherit;
   text-align: left;
   width: 115%;
   position: absolute;
   top: -7.1rem;
   right: -7.1rem;
   padding: 0.7rem;
-  box-shadow: 0px 0px 4px #808080;
+  box-shadow: 0px 0px 4px $color-gray-medium;
```

Comment:
close enough https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/52f75afbcd5e4217be513729e50a0fcc0dc92cf2/packages/formation/sass/base/_b-variables.scss#L72